static/preference.json

```diff
@@ -28,5 +30,9 @@
   "tabSize": 4,
   "listIndentation": 1,
 
-  "theme": "light"
+  "theme": "light",
+
+  "sideBarVisibility": false,
```

Comment:
I also recommend not put `sideBarVisibility `, `tabBarVisibility ` and `sourceCodeModeEnabled ` in preferences, Because they should be part of the window state, they should be placed in the session. Putting it into preferences can be misleading.

Reply:
These are just default/fallback values if no session or state is available.